src/main/java/org/rsa/logic/data/models/GuildConfiguration.java

```diff
@@ -0,0 +1,29 @@
+package org.rsa.logic.data.models;
+
+import lombok.Getter;
+import lombok.Setter;
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbBean;
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbPartitionKey;
+
+@DynamoDbBean
+@Getter
+@Setter
+public class GuildConfiguration {
+    Integer schema_version = 1;
+    String guildid; // Partition key
+
+    String upvote_emoji = "";
+    String downvote_emoji = "";
+    String moderate_emoji = "";
+
+    String moderator_role_id = "";
+
+    public GuildConfiguration() { }
+    public GuildConfiguration(String guildId) { guildid = guildId; }
+
+    @DynamoDbPartitionKey
+    public String getGuildid()
+    {
+        return guildid;
+    }
```

Comment:
Is this setup in a way that when we want to add some configuration for a guild, we add the data and handling then update the schema version?